### ISSUE TITLE
Add vllm serve example

### ISSUE THREAD

**Issue by maxjeblick** [NVIDIA/logits-processor-zoo#18]
Add a logits processor example that

1. Serves the model using `vllm serve` command
2. Uses openai client to create responses

**Motivation:**
Adding this example helps user that host models with vllm.
The current vllm documentation is a bit sparse on how to setup 1. and 2.

References: e.g. https://github.com/vllm-project/vllm/issues/16399
